storage/persistent-storage/persistent-storage-manila.adoc

```diff
@@ -0,0 +1,28 @@
+[id="persistent-storage-using-manila"]
+= Persistent storage using OpenStack Manila
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-manila
+
+toc::[]
+
+{product-title} is capable of provisioning PVs using the
+link:https://wiki.openstack.org/wiki/Manila[OpenStack Manila] shared 
+file system service.
+
+It is assumed the OpenStack Manila service has been correctly set up and is
+accessible from the {product-title} cluster. Only the NFS share types can be
```

Comment:
Is "share types" plural correct here, or should it be type singular?

Reply:
Corrected this to be singular. Good catch!